zone.rb

```diff
@@ -0,0 +1,35 @@
+class Zone
+
+  def doZone
+    size = ""
+    while @size < 3
+      if @size < 3
```

Comment:
I think this `size` condition is unnecessary because `@size` will always be `< 3` because of the while condition.